addons/crm_iap_enrich/tests/test_lead_enrich.py

```diff
@@ -56,3 +56,16 @@ def test_enrich_error_jsonrpc_exception(self):
 
         for lead in leads:
             self.assertEqual(lead.street, False)
+
+    def test_lead_enrich_auto_setting(self):
+        cron = self.env.ref('crm_iap_enrich.ir_cron_lead_enrichment', raise_if_not_found=False)
```

Comment:
Not sure if "raise_if_not_found" is useful here as the test will crash if the cron record is not found anyway.

Better have a clear error in that case, even though it should not happen.

Reply:
I made the change.
Thanks for the review.